### ISSUE TITLE
No more formatting since upgrade to 9.10.0

### ISSUE THREAD

**Issue by jenthomas88** [flyingsaucerproject/flyingsaucer#426]
I've upgraded this morning from 9.9.5 to 9.10.0, this is the only change in my application and since this upgrade all the content of my documents is shrinked at the top of the document.

I've tried with a very simple template but the problem is stil there:
![image](https://github.com/user-attachments/assets/659804d3-fc0e-40d8-b1a3-716a69d843b3)

This is with version 9.9.5:
![image](https://github.com/user-attachments/assets/e9a7b80d-480d-4cb4-98cc-5f3a0202eb31)

This is with version 9.10.0:
![image](https://github.com/user-attachments/assets/14ca72c5-50eb-4ab3-87a4-84f5c3f962b4)
yes, it is the same zoom :-)
When I select the small spot, I can see that my text is really there, just illegible.

My generation code is very simple:
```java
public void run(Path processedFile, Path completedFile, List<GeneratorFeature> features)
            throws GeneratorException {
        try (FileOutputStream baos = new FileOutputStream(completedFile.toFile())) {
            ITextRenderer renderer = ITextRenderer.fromUrl(processedFile.toUri().toString());
            renderer.layout();
            renderer.createPDF(baos);
            log.info("PDF {} generated successfully", completedFile.toFile());
        } catch (Exception e) {
            throw new GeneratorException("Error during generation", e);
        }
    }
```

Thank you!

**Comment by asolntsev** [comment]
For the record, I found that commit https://github.com/flyingsaucerproject/flyingsaucer/commit/c42e40d6 is guilty. Continue investigation...

**Comment by asolntsev** [comment]
@jenthomas88 Fixed the formatting, released 9.10.1: https://github.com/flyingsaucerproject/flyingsaucer/releases/tag/v9.10.1

**Comment by jenthomas88** [comment]
It's working, thanks!
